modules/aws_vpc/variables.tf

```diff
@@ -103,18 +103,6 @@ variable "map_public_ip_on_launch" {
   default     = true
 }
 
-variable "vpc_private_enabled" {
-  description = "Use Private IP address for cluster API endpoint"
-  type        = bool
-  default     = false
-}
-
-variable "vpc_private_endpoints" {
-   description = "Endpoints needed for private cluster"
-   type        = list(string)
-   default     = [ "ec2", "ecr.api", "ecr.dkr", "s3", "logs", "sts", "elasticloadbalancing", "autoscaling" ]
-}
```

Comment:
All of these items, as noted above, were in place to keep the infra private internally to help with security. Need to know what/how these items are now handled and locked down as stated above.